### ISSUE TITLE
enums naming generation bug

### ISSUE THREAD

**Issue by atzedus** [stephenafamo/bob#216]
1) init database:

```SQL
CREATE TYPE platform AS ENUM ('DESKTOP', 'MOBILE', 'SMART_TV');
```
2) generate code by bob v0.26.0

Expected:
```go
// Enum values for Platform
const (
	PlatformDESKTOP    Platform = "DESKTOP"
	PlatformMOBILE     Platform = "MOBILE"
	PlatformSMART_TV Platform = "SMART_TV"
)
```

result:

```go
// Enum values for Platform
const (
	PlatformDESKTOP    Platform = "DESKTOP"
	PlatformMOBILE     Platform = "MOBILE"
	PlatformSMARTU5fTV Platform = "SMART_TV"
)
```

PlatformSMARTU5fTV

**Comment by stephenafamo** [comment]
A side effect of better support for symbols 😅 

I'll fix this soon

**Comment by stephenafamo** [comment]
Should be fixed in #218. 

Try it out and let me know.
